app.py

```diff
@@ -575,55 +582,88 @@ def generate_figure(street, direction, day_type='Weekday', period='AMPK',
+                            html.Div(id=TABLE_DIV_ID, children=generate_table(INITIAL_STATE['ew'], 'Weekday', 'AM Peak')),
+                            html.Div([html.B('Travel Time', style={'background-color':'#E9A3C9'}),' 1+ min', html.B(' longer'), ' than baseline']),
+                            html.Div([html.B('Travel Time', style={'background-color':'#A1D76A'}),' 1+ min', html.B(' shorter'), ' than baseline']), 
+                                             
+                        ])
+        ])                    
+
+app.layout = html.Div([
+            dbc.Row(
+                dbc.Col([html.H1(children=TITLE, id='title')],
+                                width=12, style={'backgroundColor': "#165788"})
+                    ),
+            dbc.Row(
+                dbc.Col([html.Div(
+                        dcc.Tabs(children=[
+                                    dcc.Tab(label='East-West Streets', value='ew'),
+                                    dcc.Tab(label='North-South Streets', value='ns')],
+                                    value='ew',
+                                    id='tabs',
+                                    style={'font-weight':'bold'}))], width=12)
+                    ),
+            dbc.Container(
+                [dbc.Row([
+                    dbc.Col((                    
+                       html.Div(id=MAIN_DIV, children=[STREETS_LAYOUT])), width={"size":4, "order":1}, sm=12, xs=12, md=12, lg=4),
+                    dbc.Col(html.Div(children=[
+                                                html.H2(id=STREET_TITLE, style={'fontSize':30, 'fontWeight':'bold'}),
+                                                html.H2(id=TIME_TITLE, style={'fontSize':25}),                  
+                                                html.H2(id=STREETNAME_DIV[0], style={'fontSize':20}),
+                                                html.Div(id = GRAPHDIVS[0], children=dcc.Graph(id=GRAPHS[0])),
+                                                html.H2(id=STREETNAME_DIV[1], style={'fontSize':20}),
+                                                html.Div(id = GRAPHDIVS[1], children=dcc.Graph(id=GRAPHS[1])),
+                            html.Div(children=[
+                                    html.Div(className="box_baseline", style={'display':'inline-block', 'margin-left':"30px"}),
+                                    html.Span("Baseline  ", style={'display':'inline-block', 'margin-left':"10px"}),
+                                    html.Div(className="box_closure", style={'display':'inline-block', 'margin-left':"20px"}),
+                                    html.Span("Closure  ", style={'display':'inline-block', 'margin-left':"10px"}), 
+                                    html.Div(className="box_selected", style={'display':'inline-block', 'margin-left':"20px"}),
+                                    html.Span("Selected  ", style={'display':'inline-block', 'margin-left':"10px"}),
```

Comment:
This should be labelled as the legend.
Maybe create it above and drop it in here as a variable?